packages/gradle/src/utils/get-project-report-lines.ts

```diff
@@ -32,6 +33,38 @@ export async function getProjectReportLines(
     return [];
   }
 
+  try {
+    execSync('java --version');
+  } catch (e) {
+    throw new AggregateCreateNodesError(
+      [
+        [
+          gradlewFile,
+          new Error(
+            `Could not find Java. Please install Java and try again: https://www.java.com/en/download/help/index_installing.html. ${e.message}`
+          ),
+        ],
+      ],
+      []
+    );
+  }
+
+  try {
+    await execGradleAsync(gradlewFile, ['wrapper']);
+  } catch (e) {
+    throw new AggregateCreateNodesError(
+      [
+        [
+          gradlewFile,
+          new Error(
+            `Could not find Gradle. Please install Gradle and try again: https://gradle.org/install/. ${e.message}`
+          ),
+        ],
+      ],
+      []
+    );
+  }
```

Comment:
Can we achieve this without running extra commands?

Can we check the error that comes back from the `exec` within `execGradleAsync`?